images/channels/amazon-prime-video.svg

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
@beaujeanfilmhub Are you sure this logo is the approved one from Amazon according to their guidelines? What's the source of this?

Also, please minify this.

Reply:
This has been corrected.